frontend/packages/console-shared/src/utils/sample-utils.ts

```diff
@@ -303,6 +307,43 @@ const defaultSamples = (t: TFunction) =>
           lazyYaml: () => YAML.dump(defaultProjectAccessRoles),
           targetResource: getTargetResource(ConsoleOperatorConfigModel),
         },
+        {
+          title: t('console-shared~Add page actions'),
+          description: t(
+            'console-shared~Provides a list of all available actions on the Add page in the Developer perspective. The IDs could be used to disable individual actions.',
+          ),
+          id: 'addpage-actions',
+          snippet: true,
+          lazyYaml: () => {
+            return new Promise<string>((resolve) => {
+              const unsubscribe = subscribeToExtensions<AddAction>(
+                (extensions: LoadedExtension<AddAction>[]) => {
+                  const sortedExtensions = extensions
+                    .slice()
+                    .sort((a, b) => a.properties.id.localeCompare(b.properties.id));
+                  const yaml = sortedExtensions
+                    .map((extension) => {
+                      const { id, label, description } = extension.properties;
+                      const labelComment = t(label)
+                        .replace(/%$/, '')
```

Comment:
@christianvogt Without this line the `t(ranslated)` labels has a `%` at its end. Did you know the reason?

Reply:
@jerolimov extensions are supposed be consumed with `useExtensions` which would have automatically translated this for you. Otherwise you do need to do what you're doing. But you would need to strip out the `%` from the start and the end of the string.

I suggest you simply add a `useExtensions` call to resolve the extensions

Reply:
As discussed in Google Meet, I fixed the translation (and removed the `.replace(/%$/, '')` and added a comment why I don't use `useExtensions` at the moment.